app/views/students/_left_nav_show.html.haml

```diff
@@ -0,0 +1,6 @@
+%ul{ class: 'side-nav'}
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should end with one space before the closing brace